src/main/java/uk/ac/cam/cl/dtg/segue/api/LogEventFacade.java

```diff
@@ -98,6 +99,13 @@ public Response postLog(@Context final HttpServletRequest httpRequest, final Map
                     + TYPE_FIELDNAME + " property.").toResponse();
         }
 
+        String eventType = (String) eventJSON.get(TYPE_FIELDNAME);
+
+        if (ISAAC_LOG_EVENT_TYPES.contains(eventType) || SEGUE_LOG_EVENT_TYPES.contains(eventType)) {
```

Comment:
Perhaps to reduce the leakage of isaac stuff into segue could we combine this collection in the IsaacGuiceConfigurationModule / Segue equivalent and then dependency inject them in? This would allow us to more easily make this into a config parameter rather than hard code what we consider to be protected classes of log events